src/Makefile

```diff
@@ -270,6 +268,11 @@ endif
 	--zone $(GCLOUD_ZONE) \
 	--project $(GOOGLE_PROJECT_ID)
 
+rollback-helm:
+	# Rollback to the previous good release
+	./helm_rollback.sh
+
+deploy-helm:
```

Comment:
I can take that as a separate task, since this would require further changes at the CI templates, but it's a good point. We can break this up even further and also start using `name:` at our CI steps to make them less cryptic.